tools/constraints.txt

```diff
@@ -15,7 +15,6 @@ coverage>=4.4.2,<=7.0.5
 astroid>=2.9.0
 pylint>=2.7.0
 pyenchant>=3.0.0
-test-generator==0.1.1
```

Comment:
could you please remove `test-generator` from other `.txt` files: 
- tools\mo\requirements_dev.txt
- tests\samples_tests\smoke_tests\requirements.txt
- tests\constraints.txt

Reply:
@rkazants  Done, What I need to do next which task is ahead for me?

Reply:
@PRATHAM-SPS, the problem is that there appeared ovc unit tests that also use `generator`. Please temporarily recover `.txt` files with this module and work on replacing `generator` with `pytest`